.openshift-ci/drivers/pre-build.Dockerfile

```diff
@@ -8,8 +8,9 @@ ENV MAX_PARALLEL_BUILDS=32
 
 COPY --from=replaced-by-osci:scripts /scripts/ /scripts/
 
-RUN dnf -y install jq && \
```

Comment:
Is jq not needed here?

Reply:
No, because the `src` image is now based on `apollo-ci:collector` which already has it installed.